libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKSalesforceAnalyticsManager.m

```diff
@@ -113,32 +113,31 @@ - (instancetype) initWithUser:(SFUserAccount *) userAccount {
 
         // Reads the existing analytics policy and sets it upon initialization.
         [self readAnalyticsPolicy];
```

Comment:
`self.loggingEnabled/self.isLoggingEnabled` is just fronting the `NSUserDefaults` storage/retrieval now.  I think this call could actually go away, since the first call to `self.isLoggingEnabled` is going to read/initialize the `NSUserDefaults` data anyway.  I'll take this line out.